e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -78,7 +78,7 @@ var _ = Describe("App With Domain", func() {
 					content, err := cli.EnvInit(&client.EnvInitRequest{
 						AppName: appName,
 						EnvName: "prod",
-						Profile: "own-iad",
+						Profile: "prodEnvironmentProfile",
```

Comment:
Is this meant to be constant instead
```suggestion
						Profile: prodEnvironmentProfile,
```

Reply:
🤦